### PR TITLE
avatar done

### PR DESCRIPTION
**Pull request by deepak29-git**
please review my avatar component and let me know about changes

### CODE REVIEW

File: component/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    gap:20px;
+    border-radius:5px;
+}
+
+/* image avatar */
+
+.avatar{
+    height:50px;
+    width:50px;
+    border-radius:999px;
```
Comment (anshikaa12) [inline component/Avatar/avatar.css]: try using 50% instead in border radius

File: component/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/* avatar css */
+.avatar-container{
+    margin: auto;
+    width: 500px;
+}
+
+.box{
+    height:100px;
```
Comment (anshikaa12) [inline component/Avatar/avatar.css]: use rem units everywhere to achieve responsiveness for the future. pixels will cause you trouble

**Review by anshikaa12 — COMMENTED** [review]
Try doing the changes i suggested rest looks fine

**Comment by AmishMishra11** [comment]
Looks good just made the changes that Anshika suggested and host it on netlify for easy preview

File: component/Avatar/avatar.css (hunk 2 above)
Comment (deepak29-git) [reply component/Avatar/avatar.css]: done

File: component/Avatar/avatar.css (hunk 1 above)
Comment (deepak29-git) [reply component/Avatar/avatar.css]: done

**Pull request merged**